cocotb/_build_libs.py

```diff
@@ -46,13 +46,7 @@ def build_extension(self, ext):
             subprocess.run([
                 "install_name_tool",
                 "-add_rpath",
-                "@loader_path",
-                os.path.join(self.build_lib, ext._file_name),
-            ], check=True)
-            subprocess.run([
-                "install_name_tool",
-                "-add_rpath",
-                "@loader_path/libs",
+                sys.prefix + "/lib",
```

Comment:
Should not be better?
```suggestion
                sysconfig.get_config_var("LIBDIR"),
```

Reply:
Possibly we can add this only on `libcocotb` and remove this here?

Reply:
Yes, I am familiar with `sysconfig`.